NEWS.md

```diff
@@ -3,8 +3,10 @@
 ## general
 * relaxed the way wrappers can be nested -- the only explicitly forbidden
   combination is to wrap a tuning wrapper around another optimization wrapper
+* added logical task attribute 'is.spatial'
```

Comment:
It's great that you're doing this, but in the future please put the changes for NEWS in the PR thread -- this avoids merge conflicts with other PRs.